build-a-brain/index.rst

```diff
@@ -0,0 +1,197 @@
+********************
+How to Build a Brain
+********************
+
+|bookcover|
+The book `How to Build a Brain <book>`_
+from Oxford University Press came out in May 2013.
+It exploits the `Neural Engineering Framework (NEF) <NEF>`_
+to develop the Semantic Pointer Architecture (SPA) for cognitive modelling.
+The book uses Nengo to explain and demonstrate
+many of the central concepts for these frameworks.
+This section of the website supports the book by providing
+links to models, demos, videos, and tutorials mentioned in the book.
+
+.. seealso::
+   * `The Nengo SPA project <https://nengo.github.io/nengo_spa>`_.
```

Comment:
That page currently does not exist. Maybe we should link to the actual GitHub repo for now?